docs/guides/installation-troubleshooting.md

```
@@ -253,6 +253,51 @@ npx --ignore-existing aiox-core@latest
 npx aiox-core@2.2.0
 ```
 
+### Issue 10: "Pro activation failed: Installed Pro artifact did not create node_modules/@aiox-squads/pro"
+
+**Symptom:** The installer reaches Step 2 (Pro Content Installation), successfully authenticates the buyer email, downloads the Pro tarball — and then prints:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)